### PR TITLE
seeedsssss

### PR DESCRIPTION
**Pull request by casantadu**
Helllloo! The seeds are finished

### CODE REVIEW

**Comment by casantadu** [comment]
photos should also work

File: app/views/cars/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="position-relative">
-  <div class="card-category card-large" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('https://media.autoexpress.co.uk/image/private/s--l9NxPLCP--/v1586268650/autoexpress/2020/04/Porsche%20911%20Turbo%20S%20Cabriolet.jpg')">
+  <div class="card-category card-large" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('<%= @car.photo %>')">
```
Comment (ymik0410) [inline app/views/cars/show.html.erb:2]: Here we would need the image tag 
<%= image_tag @car.photo %>

File: cars1.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<HTML><HEAD>
```
Comment (ymik0410) [inline cars1.html]: Do we really need the files cars, cars1 and cars.htmlcurl ?

**Review by ymik0410 — COMMENTED** [review]
Seed file looks good, but I think we do not need the other files. Could you check it please?

**Pull request merged**